paddle/fluid/memory/allocation/allocator.h

```diff
@@ -19,21 +19,21 @@
 #include <utility>
 #include <vector>
 #include "paddle/fluid/framework/inlined_vector.h"
+#include "paddle/fluid/platform/enforce.h"
 #include "paddle/fluid/platform/place.h"
 
 namespace paddle {
 namespace memory {
 namespace allocation {
 
 // Exception when `Alloc`/`AllocShared` failed
-class BadAlloc : public std::exception {
- public:
-  inline explicit BadAlloc(std::string msg) : msg_(std::move(msg)) {}
+struct BadAlloc : public std::exception {
```

Comment:
I found that there are not only one place using f, l ... including old code ... please just modify the f -> file, l -> line at newly added code. You don't have to change old code.

Reply:
Done. I have changed the old codes too.